testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -5071,6 +5071,41 @@ QUnit.module('the \'fieldTemplate\' option', moduleSetup, () => {
     });
 });
 
+
+QUnit.module('the \'valueChangeEvent\' option', moduleSetup, () => {
+    ['keyup', 'blur', 'change', 'input', 'focusout'].forEach((eventName) => {
+        QUnit.test(`custom item has been added when valueChangeEvent='${eventName}'`, function(assert) {
```

Comment:
Shouldn't we also test that custom item is added to a datasource? (how many times custom item creating event was called)